src/index.ts

```diff
@@ -17,4 +18,4 @@ export const convertArrayToCSV = (data, { header, separator } = {}) => {
   return convertArrayOfObjectsToCSV(data, thisOptions);
 };
 
-export default convertArrayToCSV;
+export { convertArrayToCSV };
```

Comment:
We still need the default export and the named export

Reply:
Could you try the new changes? Everything worked for me and I've it working on a project.

Reply:
Yeah it works but we this version has a default and a named export so a major release would be required if we change that to only named. So we still need both exports.